docs/design/autoware-interfaces/ad-api/features/fail-safe.md

```diff
@@ -45,3 +47,16 @@ This is service dependent. Autoware supports the following transitions by defaul
 | COMFORTABLE_STOP | The vehicle will stop quickly with a comfortable deceleration.            |
 | EMERGENCY_STOP   | The vehicle will stop immediately with as much deceleration as possible.  |
 | PULL_OVER        | The vehicle will stop after moving to the side of the road.               |
+
+## MRM request
+
+MRM request is a feature that triggers MRM from an application, and is mainly used when the application wants to transition the vehicle to a safe state by detecting an abnormality. When an MRM request is sent, Autoware attempts to operate the MRM. To distinguish requests from multiple applications, an MRM request must include a user name as an identifier.
+
+Since MRM has multiple behaviors, this feature provides a selection strategy. Please refer to the table below for supported strategies.
+If there are multiple requests, the strategy with the highest priority will be used.
+Note that when Autoware detects an abnormality, it may not be able to operate according to the MRM request strategy.
+
+| Strategy | Description                                                |
+| -------- | ---------------------------------------------------------- |
+| CANCEL   | Cancel the MRM request associated with the specified user. |
+| DELEGATE | Delegate the selection of MRM behavior to Autoware.        |
```

Comment:
We only have cancel and delegate.
Were there any consideration about selecting MRM behavior from an application?

In other words, if an application is triggering MRM, then Autoware is not aware why it should be taking MRM behavior. Is it possible for Autoware to select the appropriate behavior in such situation? 

If this is future work, that is okay, but we might want to state that in the description.

Reply:
In the future, I will consider to add strategies that vehicles achieve MRC as quickly as possible, as comfortable as possible, or etc.